apps/api-documenter/src/documenters/OfficeYamlDocumenter.ts

```diff
@@ -48,8 +48,8 @@ export class OfficeYamlDocumenter extends YamlDocumenter {
     console.log('Loading snippets from ' + snippetsFilePath);
 
     const snippetsContent: string = FileSystem.readFile(snippetsFilePath);
-    this._snippets = yaml.load(snippetsContent, { filename: snippetsFilePath });
-    this._snippetsAll = yaml.load(snippetsContent, { filename: snippetsFilePath });
+    this._snippets = yaml.load(snippetsContent, { filename: snippetsFilePath }) as ISnippetsFile;
+    this._snippetsAll = yaml.load(snippetsContent, { filename: snippetsFilePath }) as ISnippetsFile;
```

Comment:
`load` isn't generic? What type does it return?

Reply:
In the new version, it returns `unknown` instead of `any` which requires an explicit type cast.

Arguably `as ISnippetsFile` better captures the assumption/conversion happening here, versus `<ISnippetsFile>`.